api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -60,6 +60,108 @@ The group cell's value with applied [format](/api-reference/_hidden/dxDataGridCo
 The group cell's value as it is specified in a data source.
 
 ---
+
+The format for group cells is `x: y` followed by an optional summary. `x` is the caption and `y` is the value of the column used to group data. In the following example, group cells display the column value only:
```

Comment:
Why `x:y` but not `caption: value`?
I suggest the next template customization example: `Group Column Caption: '${options.caption}' with '${options.value}'`
Because the current example doesn't demonstrate how to get column caption.